build/media_source/plg_system_guidedtours/scss/guidedtours.scss

```
@@ -0,0 +1,45 @@
+/**
+ * @copyright  (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+ @import '../../../../node_modules/shepherd.js/dist/css/shepherd';
```

```suggestion
@import "../../../../node_modules/shepherd.js/dist/css/shepherd";
```

Fixes the linter issues